apps/coscrad-frontend/src/components/resources/bibliographic-references/book-bibliographic-reference-detail.full-view.presenter.tsx

```diff
@@ -25,12 +29,24 @@ export const BookBibliographicReferenceDetailFullViewPresenter = ({
         ] as [string, string][]
     ).map(buildValueAndDisplay);
 
+    const name = title;
+
     return (
-        <BibliographicReferenceCard
+        <ResourceDetailFullViewPresenter
+            name={name}
             id={id}
-            header="Book Bibliographic Reference"
-            title={title}
-            labelsAndValues={labelsAndValues}
-        />
+            type={ResourceType.bibliographicReference}
+        >
+            <SinglePropertyPresenter display="Title" value={title} />
+            <SinglePropertyPresenter
```

Comment:
I think it's helpful to see the specific type for bibliographic references (court case, journal article, book).  Could possibly add a helper to capitalize and add a space.